docs/how-to-guides/reuse-packages-between-builds.rst

```diff
@@ -0,0 +1,16 @@
+.. tip::
+
+    This is a preview of the reusable *Reuse packages between builds* documentation.
+    It can be included in your documentation by inserting a file that looks as follows,
+    with the appropriate changes
+
+    .. literalinclude:: reuse-packages-between-builds.rst
+        :start-after: .. :docs-include:
```

Comment:
There is no blank line here because otherwise the blank line would be included in the self-referential literalinclude.